tests/remotes/gs.py

```diff
@@ -64,24 +63,63 @@ def _get_storagepath():
     def get_url():
         return "gs://" + GCP._get_storagepath()
 
-    @classmethod
-    @contextmanager
-    def remote(cls, repo):
-        yield Remote(GSRemoteTree(repo, {"url": cls.get_url()}))
+    @property
+    def _gc(self):
+        from google.cloud.storage import Client
```

Comment:
Cloud fixtures try to implement methods by itself without relying on Tree too much, so that our tests are separated from our code. But sometimes it is just too hard to do that so in stuff like ssh, we use SSHConnection for now, but I might switch to plain paramiko after all, to make it more isolated.